DigitalLearningSolutions.Web/Scripts/trackingSystem/courseDelegates.ts

```diff
@@ -7,7 +7,7 @@ const courseSelectBox = <HTMLSelectElement>document.getElementById('selected-cus
 const customisationId = courseSelectBox?.value.trim();
 
 const exportCurrentLink = <HTMLAnchorElement>document.getElementById('export-current');
-exportCurrentLink.addEventListener('click', (e) => {
+exportCurrentLink.addEventListener('click', () => {

```

Comment:
removed this because it gave me an eslint error, even though I didn't touch this file